spec/requests/admin_claim_checks_spec.rb

```diff
@@ -24,7 +24,20 @@
         end
       end
 
-      context "when claim is already approved" do
+      it "can reject a claim" do
+        freeze_time do
```

Comment:
I think these `freeze_time` calls are a hangover from when we were explicitly testing against an `approved_at` column. Because we're now relying on the timestamp of the check, I think we can lose them without impacting the spec.